### PR TITLE
new speak-clearly-llc-python-telephony-developer

### PR DESCRIPTION
**Pull request by jsalsman**
Not sure why the CI task for speak-clearly-llc-python-telephony-developer.html failed; resubmitting PR.

### CODE REVIEW

**Comment by salimfadhley** [comment]
Do you need some help with this? It looks like your data is not valid. The first error was caused by trailing tab characters on the tags. The second error appears to be that you've changed the structure of the data in the header - you need to use the exact format given in the example.

File: jobs/speak-clearly-llc-python-telephony-developer.html [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
+  - Twilio
+  - Flask
+  - Javascript
+  - WebRTC
+  - C/C++
+  - PostgreSQL
+  - Git
+  - GitLab CI
+  - Docker
+  - Cross-validation
```
Comment (salimfadhley) [inline jobs/speak-clearly-llc-python-telephony-developer.html]: I'm not sure that cross-validation is a useful tag here. Are there any synonyms for this you might use?

File: jobs/speak-clearly-llc-python-telephony-developer.html [hunk 2]
```diff
@@ -10,17 +10,17 @@
...
-  - Git	
-  - GitLab CI	
-  - Docker	
-  - Cross-validation	
+  - Python
+  - Twilio
+  - Flask
+  - Javascript
+  - WebRTC
+  - C/C++
```
Comment (salimfadhley) [inline jobs/speak-clearly-llc-python-telephony-developer.html]: "c" and "c++" should be two separate tags, but do you really need both?

File: jobs/speak-clearly-llc-python-telephony-developer.html [hunk 3]
```diff
@@ -10,17 +10,17 @@
...
+  - PostgreSQL
+  - Git
+  - GitLab CI
```
Comment (salimfadhley) [inline jobs/speak-clearly-llc-python-telephony-developer.html]: Just "gitlab" will do. CI is just one small part of GitLab.

File: jobs/speak-clearly-llc-python-telephony-developer.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
+---
+title: Python telephony developer
+company: Speak Clearly, LLC
+url: https://bit.ly/twiliojob
+location: Remote
+contract: perm
+contact:
+    name: Jim Salsman
+    email: jim@talknicer.com
+    ghangouts: jsalsman
```
Comment (salimfadhley) [inline jobs/speak-clearly-llc-python-telephony-developer.html]: "ghangouts" is not valid.

File: jobs/speak-clearly-llc-python-telephony-developer.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+title: Python telephony developer
```
Comment (salimfadhley) [inline jobs/speak-clearly-llc-python-telephony-developer.html]: "Telephony Developer" would be a more appropriate title, given that all jobs on this site are Python.

File: jobs/speak-clearly-llc-python-telephony-developer.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+url: https://bit.ly/twiliojob
+location: Remote
+contract: perm
+contact:
+    name: Jim Salsman
+    email: jim@talknicer.com
+    ghangouts: jsalsman
+created: !!timestamp 2019-07-09
+tags: 
+  - Python
```
Comment (salimfadhley) [inline jobs/speak-clearly-llc-python-telephony-developer.html]: You do not need Python as a tag on the Python jobs board.

Comment (jsalsman) [reply jobs/speak-clearly-llc-python-telephony-developer.html]: @salimfadhley thank you! edited: https://github.com/pythonjobs/jobs/pull/385/commits/3f73286f5245dcebd10bbfc3a6d45eccf5722b6a

**Comment by jsalsman** [comment]
I fixed additional issues with the keywords: MSVC++ and https://github.com/jsalsman/jobs/commit/7364b21cdfdd4c6bae3ed242fd732ce36f7ea92b#diff-828cf715bd9db34fd22348dc6e7c7cdd cc @salimfadhley thank you for your help.

**Pull request merged**